### DISCUSSION TITLE
Sibionics CGM / Companion APP

### DISCUSSION THREAD

**Discussion by sfriedriszik** [NightscoutFoundation/xDrip#3063] [Ideas]
Would it be possible to read out the Sibionics CGM using the Companion App function?
I am available for questions, I am wearing such a sensor at the moment.

http://en.sibionics.com/Consumer/index.html

Regards Stefan

**Comment by jamorham** [comment]
Does the app show glucose values via a notification?

**Reply by jrkf** [reply]
Notifications are aviable only on low or high bg

**Reply by jrkf** [reply]
I can get bg data from sibionics cloud, but can you tell me how to broadcast data to xdrip or aaps?

**Reply by jrkf** [reply]
Ok I made it! :) I can see BG on xdrip and aaps from sybionics cgm

**Reply by Giuseppe6479** [reply]
Hi, could you please explain the procedure to see BG on xdrip and aaps from sybionics cgm? Thank you

**Reply by jrkf** [reply]
It is no eazy but i can try:
1. Install pcapDroid app on your phone
2. Scan sybionics app to find https reqest to get your BG
3. Use tasker or automate app to build request every 5 min and put data using broadcast to xdrip

If you are interested about details let me known

**Reply by itudor2** [reply]
Hi jrkf, can you give more details, please? Thx.

**Reply by jrkf** [reply]
**To get BG from sybiocics cloud:**

GET req on https://cgm-ce.sisensing.com/user/app/follow/sharer
Headers: Authorization: token

Token you can find using pcapDroid  app

response:
{"timestamp":1696611684219,"code":200,"msg":"Success","data":[{"id":"1701288253057011994","userId":"1701244402032316746","userEmail":"xxx@gmail.com","userNickname":"xxx","deviceId":1701254241957581578,"enableTime":"1694445607345","lastTime":"1695655207345","status":2,"latestIndex":20160,"**latestValue**":**9.20**,"latestDataStatus":-1,"alarmStatus":1,"abnormalTime":null,"latestTime":"1695655147345","alarm":{"enable":true,"style":2,"upper":180.2,"lower":70.3,"sound":"","alarmInterval":5,"forceRemind":null}}],"errorData":null,"success":true}

**To send BG to xdrip**

Broadcast send:
Package: "com.eveningoutpost.dexdrip"
Action: com.microtechmd.cgms.aidex.action.BgEstimate
Extras: {"com.microtechmd.cgms.aidex.SensorId" as String: time, "com.microtechmd.cgms.aidex.Time" as Long: time2, "com.microtechmd.cgms.aidex.BgType": "mg/dl", "com.microtechmd.cgms.aidex.BgValue" as Double: round(bg)}

**Reply by LeoBounce** [reply]
Can you please help with details on how to get a token and how to add these rules to Tasker? On xDrip+ it is said that SIBionics is supported via Companion, but SIBionics does not show any sugar in notifications, so without advanced configuration (like your method) it is not possible. I'm using the official Chinese version of SiBionics and need to broadcast values to xDrip+
Thank you in advance!

**Reply by jrkf** [reply]
Hi Leo as I mention before install pcapDroid app on your phone and add sibionics app to scan https connection. Then you will find token

**Reply by jrkf** [reply]
Ok, my friend help me to create documentation how to connect sybionics cgm with Xdrip.
Below you can find this doc -> https://drive.google.com/file/d/1iqOlV91P5OlaiCRY4FLsiivZZ-dDU4Du/view?usp=drivesdk

And flow ->
https://drive.google.com/file/d/1it0B2_rBRGXCyQFwzj2PCzsCqmszXYQ2/view?usp=drivesdk

**Reply by LeoBounce** [reply]
Hi jkrf,
Thank you! It works well! In my case, I changed the Automate graph to make it work with the Chinese version of SiBionics, which has a different URL, JSON format, and variable names.
I attached the CN version graph. The rest of the steps are similar.
[SiBionics CN.zip](https://github.com/NightscoutFoundation/xDrip/files/13433091/SiBionics.CN.zip)

**Reply by yiyoMX** [reply]
Does this only works with an internet connection?

**Reply by jrkf** [reply]
Yes

**Reply by fredericogcs** [reply]
Hey guys, any chance there is an alternative for the same issue - getting sibionics readings into noghtscout - but for iphone user? Many thanks in advance

**Reply by waynez** [reply]
I believe you can use iOS Shortcuts to do the same as Automate on Android. Basically you just get the glucose data from Sibionics server and send it to nightscout, that's what it's really about.

In fact, you don't really have to work on a mobile phone. As long as you have the token to make API calls, you can just do some scripting and run it wherever you like, like a desktop.

**Reply by ImLovinIt** [reply]
If anyone is looking for a different way to upload the SiBIonics glucose data to Nightscout, I have written a python script and dockerised it for deployment. It is currently tested with the CN app and API server only. Feel free to fork/clone and tweak it.

Here is the github repo.
https://github.com/ImLovinIt/Nightscout-SisensingCGM-Uploader

**Reply by ImLovinIt** [reply]
> Hi jkrf, Thank you! It works well! In my case, I changed the Automate graph to make it work with the Chinese version of SiBionics, which has a different URL, JSON format, and variable names. I attached the CN version graph. The rest of the steps are similar. [SiBionics CN.zip](https://github.com/NightscoutFoundation/xDrip/files/13433091/SiBionics.CN.zip)

Hi LeoBounce,
Can you shed some light on how you find the URL? When I scan the app on my iOS phone (connecting to the sensor), it never requests from `https://api.sisensing.com/follow/app/follow/myself/glucose/details/devices`. Therefore I don’t get `glucoseInfos` in a json response. However, I have got some success with a follower phone. The URL will have unique follower ID tho. 
Siobionics just released a 18-months model with a different lite app. I’d like to find out its API request URL without involving follower mode.
Thanks

**Reply by Navid200** [reply]
Guys,

With all due respect, we have added the ability to get readings from the Sibionics app into xDrip.
If that is not working for you, we can talk about that here.

If you need something else or if you have come up with a scheme to do anything clever useful to everyone, as long as it is not directly related to xDrip, please take the conversation to a different platform.

This repository contains valuable data for many worldwide who use xDrip.  I am one of them and I need this repository to survive.  If some day someone decides to attack xDrip and shut it down, I don't want any conversation in these discussions to be shown as evidence for xDrip being a place where people talk about hacking things.  That is not the right image for xDrip.
I use xDrip myself and I need it to be there in years to come for me and everyone else who uses it.

xDrip has very limited resources.  Please help us focus on what xDrip is about and nothing else in this repository.

Thanks for your help.

**Reply by ImLovinIt** [reply]
> Guys,
> 
> With all due respect, we have added the ability to get readings from the Sibionics app into xDrip. If that is not working for you, we can talk about that here.
> 
> If you need something else or if you have come up with a scheme to do anything clever useful to everyone, as long as it is not directly related to xDrip, please take the conversation to a different platform.
> 
> This repository contains valuable data for many worldwide who use xDrip. I am one of them and I need this repository to survive. If some day someone decides to attack xDrip and shut it down, I don't want any conversation in these discussions to be shown as evidence for xDrip being a place where people talk about hacking things. That is not the right image for xDrip. I use xDrip myself and I need it to be there in years to come for me and everyone else who uses it.
> 
> xDrip has very limited resources. Please help us focus on what xDrip is about and nothing else in this repository.
> 
> Thanks for your help.

Hi,
Understand. I will take it offline.

**Reply by thiagomsoares** [reply]
> Guys,
> 
> With all due respect, we have added the ability to get readings from the Sibionics app into xDrip. If that is not working for you, we can talk about that here.
> 
> If you need something else or if you have come up with a scheme to do anything clever useful to everyone, as long as it is not directly related to xDrip, please take the conversation to a different platform.
> 
> This repository contains valuable data for many worldwide who use xDrip. I am one of them and I need this repository to survive. If some day someone decides to attack xDrip and shut it down, I don't want any conversation in these discussions to be shown as evidence for xDrip being a place where people talk about hacking things. That is not the right image for xDrip. I use xDrip myself and I need it to be there in years to come for me and everyone else who uses it.
> 
> xDrip has very limited resources. Please help us focus on what xDrip is about and nothing else in this repository.
> 
> Thanks for your help.

Hello, and how did this integration turn out? I couldn't find it.

**Reply by Navid200** [reply]
Follow this to get readings in xDrip:
https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html

**Reply by BBQ873** [reply]
> Guys,
> 
> With all due respect, we have added the ability to get readings from the Sibionics app into xDrip. If that is not working for you, we can talk about that here.
> 
> If you need something else or if you have come up with a scheme to do anything clever useful to everyone, as long as it is not directly related to xDrip, please take the conversation to a different platform.
> 
> This repository contains valuable data for many worldwide who use xDrip. I am one of them and I need this repository to survive. If some day someone decides to attack xDrip and shut it down, I don't want any conversation in these discussions to be shown as evidence for xDrip being a place where people talk about hacking things. That is not the right image for xDrip. I use xDrip myself and I need it to be there in years to come for me and everyone else who uses it.
> 
> xDrip has very limited resources. Please help us focus on what xDrip is about and nothing else in this repository.
> 
> Thanks for your help.

Hi, can you tell me the current best way to pull readings from the Sibionics app into xDrip? The docs mention a patched app (Sugar joy+ I believe), but from what I’ve seen, it’s now restricted to Russian users. Others also mentioned the companion app, but I found it not working. Is there another method that works now?  Thanks!

**Reply by Navid200** [reply]
I don't believe we have the companion app method working for this as I am told the notification the official app shows does not include the current reading, which is a requirement for the xDrip companion app mode.  

I am sorry about my incorrect statement.

**Reply by BBQ873** [reply]
Thanks anyways

**Comment by itudor2** [comment]
It works with the modified app Sugar Joy+ that can be downloaded from here https://t.me/kubarev_modz
In xDrip set the data source to 640g/Eversense.

**Reply by psonnera** [reply]
Thanks, I thought the feature was embedded in the vendor app.

**Reply by yiyoMX** [reply]
Hi, with this app you don't have to do anything else but set the source to 640g/Eversense?

**Reply by Inkalika** [reply]
Yes. to calibrate, enable the calibration plugin

**Reply by yiyoMX** [reply]
The app only shows the data in mmol/L, there is no way of changing this, also alot of the text is in Russian. I wish I knew this before starting a sensor.

**Reply by yiyoMX** [reply]
I was able to install the official app and I could add the sensor even though it was already started.

**Reply by yiyoMX** [reply]
This was my mistake, there is an option to change from mmo/L to mg/dL.

**Reply by yiyoMX** [reply]
Im using it now with xdrip, why would i want to calibrate if the data that the patched app sends to xdrips supposedly is already factory calibrated?

**Reply by VitaminNN2sdsd** [reply]
"It works with the modified app Sugar Joy+" - it's not work now. Author added in app the check key for every sensor and sell keys in Russia only.

**Reply by aptash** [reply]
Sugarjoy+ works as it worked before. No changes to this app.

The problem with SugarJoy+ is:
The author of SugarJoy+ says that manufacturer modified Sibionics sensors in a way that them will not work with old versions of app. SugarJoy+ is a mod of SiBionics app 1.0.3 version.

The author created a new app. The new app checks a key for every sensor activation. The key isn't for sale but you have to buy a sensor from certain list of sellers to use the new app.

**Reply by gerkensm** [reply]
Does anyone still have the APK? As far as I can tell, the distribution through the telegram channel has stopped as the people behind it want to sell their sensors instead.

**Reply by VitaminNN2sdsd** [reply]
Hi All! 
You may try new App for Sibionics: Juggluco. Removed another 15 days limit on sensor age.

**Reply by LeoBounce** [reply]
Thanks, it works and can be broadcasted to xDrip. But the values are very different from the SiBionics app - much higher

**Reply by yiyoMX** [reply]
There is no reference to Sibionics on Juggluco github, where can I download it?

**Comment by Navid200** [comment]
@psonnera

**Comment by HeckfyCU** [comment]
Is it possible to integrate support for sensors directly in xdrip? It's cheaper than libre 3 and can work for 24 days

**Reply by psonnera** [reply]
The vendor app does broadcast to xDrip+.

**Reply by qq254344379** [reply]
Is it possible to assemble directly to xd

**Reply by Inkalika** [reply]
This sensor has problems when working with xDrip. Since it is necessary to enable the active plugin, bugs appear:
1) Raw readings are sent to Nightscout, not calibrated ones (after using the plugin)
2) If you delete calibrations on the phone Master or do calibrations. then instead of the sugar level, followers will display “???” until the initial calibration is completed. If 2 followers are enabled via the xdrip follower source, then this error can occur at any time and can be corrected by deleting one of the followers.

**Reply by yiyoMX** [reply]
From what I've read this sensor is on par with xdrip on accuracy, and it's way cheaper. Aidex on the other hand I've read comments that it's pretty bad like 20% MARD bad and that one is integrated with xdrip. Sibionics is a must for xdrip.

**Reply by Navid200** [reply]
What do you mean this sensor is on par with xDrip?
xDrip is not a sensor.

**Reply by Inkalika** [reply]
The sensor itself is as accurate as Libre 1, it needs to be calibrated. In the first days it decreases very significantly (up to 3 mmol)

**Reply by yiyoMX** [reply]
Why does it needs to be calibrated on xdrip but with the Sibionics app doesn't?

**Reply by yiyoMX** [reply]
@Navid200 can you help me with this question?

**Reply by Navid200** [reply]
No, I have no experience with Sibionics.

**Comment by yiyoMX** [comment]
I meant Dexcom.

**Comment by yiyoMX** [comment]
@Navid200 Hi, why does the documentation says: "Both the vendor app and the patched app will allow you to use xDrip+ with the [640G/Eversense](https://xdrip.readthedocs.io/en/latest/install/640_670) data source."? Does it works with the official vendor app?

**Reply by yiyoMX** [reply]
I'm talking about Sibionics.

**Reply by Navid200** [reply]
@psonnera 

I only document what I have tested: https://navid200.github.io/xDrip/
The exception to that is something we have been asked to document by the lead developer, like the web follower.
I have no way of testing Sibionics.

**Reply by yiyoMX** [reply]
I mean in the official documentation: https://xdrip.readthedocs.io/en/latest/install/datasource/#sibionics
This is the official documentation, right?

**Reply by psonnera** [reply]
There is no official documentation. Sibionics removed the broadcast. You need the patched app.

**Reply by Navid200** [reply]
I document what I know.  Others document what they know.
Putting all the documentation in one place is a mistake in my opinion and will only lead to outdated documentation sooner or later.   
I will avoid such an arrangement at all cost.

You would be right if xDrip was a company and we had a documentation department.  That's not the case.

**Reply by psonnera** [reply]
The rest of the community collaborates on documentation. xDrip+ doesn't. Neither Loop not AAPS not iAPS not Nightscout have a documentation department. Like all other open source apps they have volunteers collaborating with developers on the documentation. xDrip+ doesn't. xDrip+ contains outdated code and only the original developers might work on it. And this doesn't prevent xDrip+ to be available.

**Reply by psonnera** [reply]
@yiyoMX anyway the answer is above https://github.com/NightscoutFoundation/xDrip/discussions/3063#discussioncomment-7189401

**Reply by LeoBounce** [reply]
It does not work on the official app. The only way to make work official Sibionics apps with xDrip (both English and latest Chinese versions) - is by using the above method by jrkf (and later in the topic, I provided an Automate graph for the Chinese version if needed). I have used that method for around a month now, and it works very well; you can use it to load data on any client without even using Nightscout. You can autotranslate that document; the instructions are very detailed. Only in the last step will you load one of the provided graph files depending on your version of Sibionic and change the "token" variable inside the graph to your unique token acquired from the HTTPS response.

**Reply by yiyoMX** [reply]
Why dont you use the patched app?, since it also works offline. Cause ie airplanes could be a problem with the online method.

**Reply by LeoBounce** [reply]
Because of subscription usage from SiBionics. In China, various subscription plans are available, and you can get sensors for the whole year at a much cheaper price, plus free test strips. Both sensors and test strips are controlled inside the app, so after you use several sensors or 40 test strips, they automatically send you additional supplies. So, the only way to use it is to keep an updated official version. Now, it is more like a complete infrastructure where all data is synced in one app (CGM, finger tests from their blood glucose meter, insulin from their pen), and all this data probably can be loaded from their server, same as CGM data. Anyway, without a subscription, I would stick with a patched version.

**Reply by yiyoMX** [reply]
@LeoBounce Do you know if the patched app is the one that is available on Telegram from https://t.me/kubarev_modz? Is this the only one? Is it safe?

**Reply by LeoBounce** [reply]
It is safe and allows extended sensor wear; no internet connection is required, and it is recommended for those who use sensors outside of China. I used it for some time before switching back to official due to the mentioned reason.

**Reply by thejazzs** [reply]
> It does not work on the official app. The only way to make work official Sibionics apps with xDrip (both English and latest Chinese versions) - is by using the above method by jrkf (and later in the topic, I provided an Automate graph for the Chinese version if needed). I have used that method for around a month now, and it works very well; you can use it to load data on any client without even using Nightscout. You can autotranslate that document; the instructions are very detailed. Only in the last step will you load one of the provided graph files depending on your version of Sibionic and change the "token" variable inside the graph to your unique token acquired from the HTTPS response.

Hello LeoBounce,

The graph file is *.flo file ? And is it using at the "Automate" app? 
Automate app is a bit complicated in case. 

And 1 more question, does Automate app has to work always at backside to get the values to Xdrip?

**Reply by LeoBounce** [reply]
Yes, all yes. The Automate app is twisted, but with a complete graph, you only have to modify a value inside a mentioned node. It still works as of today.

**Reply by thejazzs** [reply]
Thank you for your kind reply. 
Yes it is working very smoothly.

I wish there was an option for Sibionics in xdrip or another app and these settings were easier to make. As far as I know, Sibionics has over 1 million users and this number is not small at all. The reason I did these operations was to transfer my glucose values to the watch in a healthy way. And this way I was able to transfer it to my watch. Thank you for your information.

**Comment by Navid200** [comment]
@psonnera 
If this is a criticism of the xDrip team, it is extremely selfish.  Anyone who criticizes a volunteer needs to get his/her priorities sorted out.

If this is a criticism of the xDrip code, (outdated code?) you have access to the code and the developers.  Please step forward and improve the code.  No one has stopped you.   I'm sure there is no outdated code in Nightscout, right?!!
Anyway, pick up a computer and code instead of criticizing those who are doing it.

If this is meant to be a criticism of me not wanting to document matters that I don't have experience in, that is OK.  I have chosen the approach that I believe is best for the users.    I am volunteer and am doing my best.  I am open to feedback and always welcome it.
But, in that case, please focus on the matter of documentation instead of stating outdated code.
I am more than happy to talk about it in the xDrip documentation room or in public.  There is nothing confidential about it.
I can point you to outdated material in every one of those you have mentioned.

**Reply by yiyoMX** [reply]
We all are grateful for the work that all the developers involved have done.

**Reply by Navid200** [reply]
I need to add something here to hopefully clarify something.

I am not the voice of xDrip.  What I state is my personal opinion.  I am a volunteer who uses xDrip and contributes to it.  I love to contribute to xDrip.
I am present here and you may see my name a lot.  But, PLEASE don't think that what I state is an official xDrip statement.

**Comment by ffernandezro** [comment]
Hello All!!

Would it be possible to read out the New Simplera Medtronic CGM using the Companion App function?
I was wearing Guardian G4 and Xdrip+ and it works really fine.

The past Monday, I changed the app from Guardian To Simplera App and the new CGM, and now Xdrip is not able to read the gluclose through the notification bar.

https://news.medtronic.com/2023-09-21-Medtronic-Diabetes-announces-CE-Mark-for-new-Simplera-TM-CGM-with-disposable-all-in-one-design

Thank you all for your valuable help!

**Comment by yiyoMX** [comment]
There is a new SIBIONICS offical app, its on the app store:
https://play.google.com/store/apps/details?id=com.sisensing.sijoy
https://apps.apple.com/es/app/sibionics/id1629392010
Does anyone know if this works now without modding with xDrip?

**Reply by aptash** [reply]
Why don't you try?
You can always switch between apps.

**Reply by twistline** [reply]
I can assume that it will never work with Xdrip, it’s unlikely that the company is interested in this.

**Reply by yiyoMX** [reply]
They don't have to support xdrip directly, if they for example have a notification with the value, it can be integrated into xdrip.

**Comment by sebastianhduy** [comment]
I’m trying the GS1 for the first time, and I hate the app. I’ve seen that you mention xDrip a lot, but not xDrip4iOS. Is it possible to connect this app to GS1?

**Comment by yiyoMX** [comment]
I see it now, can you start a sensor that already has been started?

**Comment by Navid200** [comment]
We can add a companion app option for this.
You will not be able to calibrate.  And I cannot guarantee that it would even work as I have not seen an official app notification.

**Reply by thejazzs** [reply]
Hello,

Unfortunately, Sibionics' companion app doesn't provide mg/dl information in phone notifications. Because of this, the companion app option in xDrip+ isn't working.

The only option seems to be the "jrkf" solution, which I've also implemented and got working.

The only place where data is shared is by installing the Sibionics' companion app on another phone and running it as a follower. If xDrip+ could somehow connect to Sibionics follower data (like LibreLinkUp), I think glucose information could be directly displayed in xDrip+.

Of course, I have no idea how this could be technically possible. Could any work be done on this or any other idea that may work?

**Reply by Navid200** [reply]
You mean it doesn't work for you, right?  I mean there are people who use mmol/L.  
There are so many ways to deal with this.  For example, you can install a variant and use it as a follower of the companion xDrip.  Then, you will have full control on the unit in the follower.
We can even add the option of changing the unit.  I'm not saying that will be done.  I'm saying it seems to me you are jumping the gun.  
I know what I have suggested is not very easy.  But, isn't it easier than the alternative.

**Reply by thejazzs** [reply]
No, it doesn't show any glucose value at notifications as mg or mmol, only notification mentions '"sibionics is working". Just when you have high or lows yes it is seen at the notification area of the phone. Even we can have full control at follower, there is no notification at the follower's phone too.
Shortly no notifications sharing via the companion app.
Did you check jrkf's solution?
Maybe there would be a way to use his solution or a part of his solution by easier way for sibionics sensor users.

**Reply by aptash** [reply]
Why don't you just use the SugarJoy+ app?

**Reply by thejazzs** [reply]
As far as I know sugar joy+ is using by special qr code which is selling in Russia and some special sellers. I am using EU version of Sibionics (not Chinese ver.). And as I know (please correct me if I am wrong), it s in Russian & Chinese languages as mixed.
I found sugar Joy's alternative at telegram Kubarev's channel which was quite difficult to figure it out. For sugar joy, I couldn't find at all, but read many docs about it, which I mentioned some above.

**Reply by thejazzs** [reply]
@aptash 
Can you please clarify the below questions ;
- Can Sugar Joy+ be freely used with any Sibionics sensors, both Chinese and EU versions?
- Do we have to buy a sensor from certain list of sellers to use the app or is it useable for all Sibionic users internationally?
- By sugar joy+ is it possible to calibrate the sensor?
- Is this app. produced for commercial purposes or for the benefit of all people with type 1 diabetes?

I will be waiting your kind reply to clarify the questions on sugar joy+ , thank you.

**Reply by aptash** [reply]
With Sugar joy+ you don't need to buy from some special sellers. It was the first App to use with Sibionics -> xDrip. But after that they decided to put some restrictions on users tie up users to sellers. But SJ+ was still working anyway. Though I'm not sure about EU version of sensors.

- it's free but not sure about working with EU
- SJ+ is just a mod of some old version of official Sibionics app. It's free.
- yes

Why don't you just use Juggluco then?
https://www.juggluco.nl/Juggluco/download.html
`New in Juggluco 9.3.2:
Should now work with EU Sibionics and Hematonix sensors;`

**Reply by thejazzs** [reply]
Thank you for your kind & detailed reply. Helped a lot. 

Do you have any idea about ; 
For Juggluco, do we have to connect a new sensor and use only Juggluco instead of Sibionic's companion app ? 
Or it can be connected even while we are using the companion app.?

**Reply by aptash** [reply]
For Juggluco you don't have to use Sibionic's companion app.
Use 640G/Eversense as source in xDrip. 
If you use AndroidAPS then you can use Juggluco without xDrip. You can push data directly to aaps from Juggluco.
Only 1 app can be connected to a sensor - Juggluco or SJ+ or some other app.

**Reply by thejazzs** [reply]
Can I use juggluco itself only? Without xdrip or androidAPS. I would like to send data to my watch. And if possible to extend the sensor. 

And one more question ; The only way to calibrate the Sibionics is Sugar Joy+ or is there any other way? I am thinking that It needs to be calibrated.

**Reply by thejazzs** [reply]
@aptash
Hello, 

I started a sibionics sensor with juggluco. Juggluco is reading glucose values per minute instead of per 5 minutes. Does it reduce the life of the sensor? And do you know is there any way to change Juggluco's reading to 5 minutes?
I will be waiting your kind reply, thank you.

**Reply by aptash** [reply]
Hi!
> Does it reduce the life of the sensor?

No. As fas as I know.

> And do you know is there any way to change Juggluco's reading to 5 minutes?

I don't know. But I doubt it. Please read the Juggluco's manual.

**Reply by thejazzs** [reply]
Hello,

What I setup as follows ;

- Started Sibionics sensor via Juggluco. 
- It is reading bg as per minute, arranged to use the sensor for approx 24 days. It is almost automatically selected.
- Connected to Xdrip+ from Juggluco as you mentioned and by that I can be able to make calibration. 
- Xdrip+'s bg readings are per 5 minutes, however, it is not making a big problem.
- Connected watch via Xdrip+ to get the calibrated results on watch.

This is the system I make for now. I will keep you informed of 24 days usage.

Thank you aptash for your kind intention. Helped a lot.

**Reply by ard3ya** [reply]
im startin to use sibionics too. i want to use it for 24 days. but struggling to do cuz cant find any content for help. i would appreciate the help if someone could.

**Comment by tkalfaoglu** [comment]
I just started with Sibionics CGM, very happy with it - its own program is better than most others.
But how did you guys get 24 days out of it? Mine stopped working after 14..
I just installed xdrip and will be setting it up as "companion" - I hope I can get it to work :)
Many thanks to everyone that made Xdrip+ possible.

**Reply by thejazzs** [reply]
Dear Turgut,

I am also Turkish by the way. Nice to meet you.
There is only one way to use it 23.5 days , which I am using at the moment and so very happy and satisfy with it.
Use Juggluco as main software and If you wish, you can connect from Juggluco to Xdrip+ either. 
By that Juggluco will send the data to Xdrip+ immediately and they can work similtaneously. 
When you start the Sibionics CGM with Juggluco, It will start giving the valus in 10 minutes and work for 23.5 days.
This time is the life time of the battery on the sensor. Even it gives better results after 14th day. 

I am using Juggluco and Xdrip together. I was using Xdrip+ due to its calibration abilities. However, right now I don't use its calibration setup because sensor itself is giving very accurate results. 

I can help you for your further questions in case, please reply to me as direct msg.
Hope It helps.

**Reply by ard3ya** [reply]
i cant connect my sibionics gs1 to juggluco. is there any spesific conditions or something?

**Reply by thejazzs** [reply]
> i cant connect my sibionics gs1 to juggluco. is there any spesific conditions or something?

What is the exact problem while trying to connect?
Use fresh sensor instead of active one. If there is QR code reading problem, take the qr code's photo by another phone, zoom it and then try to read the qr code again, mostly it is the problem facing and this is the solution.
Or else please clarify the detail of the problem in case to proceed further.

**Reply by ard3ya** [reply]
like i couldnt connect my last one to juggluco, yeah it was active one and there was qr reading problem. ill try the methods you mentioned. thanks a lot.

**Reply by tkalfaoglu** [reply]
> Dear Turgut,
> 
> I am also Turkish by the way. Nice to meet you. There is only one way to use it 23.5 days , which I am using at the moment and so very happy and satisfy with it. Use Juggluco as main software and If you wish, you can connect from Juggluco to Xdrip+ either. By that Juggluco will send the data to Xdrip+ immediately and they can work similtaneously. When you start the Sibionics CGM with Juggluco, It will start giving the valus in 10 minutes and work for 23.5 days. This time is the life time of the battery on the sensor. Even it gives better results after 14th day.
> 
> I am using Juggluco and Xdrip together. I was using Xdrip+ due to its calibration abilities. However, right now I don't use its calibration setup because sensor itself is giving very accurate results.
> 
> I can help you for your further questions in case, please reply to me as direct msg. Hope It helps.

Hi! You had a guide and I lost it. can you send it again if possible? :)  many thanks!

**Comment by BBQ873** [comment]
来信收到，谢谢!

**Comment by BBQ873** [comment]
来信收到，谢谢!
